### PR TITLE
Implemented User mentions in textfield

### PR DESCRIPTION
**Pull request by alecxchuk**
@pauleke65 @jeremiahseun @YenumJay

### CODE REVIEW

**Review by YenumJay — CHANGES_REQUESTED** [review]
Not sure what was done. Please rename your PR and add a description.

File: lib/ui/view/expandable_textfield/expandable_textfield_screen_viewmodel.dart [hunk 1]
```diff
@@ -55,4 +64,40 @@ class ExpandableTextFieldScreenViewModel extends BaseViewModel {
...
+
+  Future<void> userMentions() async {
+    setBusy(true);
+    matchedUsers = channelUsersList =
+        await channelsApiService.getChannelMembers("615045e9cf2c0f1ad758558d");
+    setBusy(false);
+    notifyListeners();
+  }
+
+  // void showMembersList(bool show) {
```
Comment (jeremiahseun) [inline lib/ui/view/expandable_textfield/expandable_textfield_screen_viewmodel.dart:76]: Please remove commented code

**Review by jeremiahseun — CHANGES_REQUESTED** [review]
Please make changes and we are good to go

**Review by jeremiahseun — APPROVED** [review]
LGTM

**Review by YenumJay — APPROVED** [review]

**Pull request merged**